atest/robot/libdoc/datatypes_json-xml.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Resource          libdoc_resource.robot
+Suite Setup       Run Libdoc And Parse Output    ${TESTDATADIR}/DataTypesLibrary.json
+
+*** Test Cases ***
+Check DataType Enums
+    DataType Enums Should Be    0
+    ...    AssertionOperator
+    ...    <p>This is some Doc</p>\n<p>This has was defined by assigning to __doc__.</p>
+    ...    {"name": "equal","value": "=="}
+    ...    {"name": "==","value": "=="}
+    ...    {"name": "should be","value": "=="}
+    ...    {"name": "inequal","value": "!="}
+    ...    {"name": "!=","value": "!="}
+    ...    {"name": "should not be","value": "!="}
+    ...    {"name": "less than","value": "<"}
+    ...    {"name": "<","value": "<"}
+    ...    {"name": "greater than","value": ">"}
+    ...    {"name": ">","value": ">"}
+    ...    {"name": "<=","value": "<="}
+    ...    {"name": ">=","value": ">="}
+    ...    {"name": "contains","value": "*="}
+    ...    {"name": "*=","value": "*="}
+    ...    {"name": "starts","value": "^="}
+    ...    {"name": "^=","value": "^="}
+    ...    {"name": "should start with","value": "^="}
+    ...    {"name": "ends","value": "$="}
+    ...    {"name": "should end with","value": "$="}
+    ...    {"name": "$=","value": "$="}
+    ...    {"name": "matches","value": "$"}
+    ...    {"name": "validate","value": "validate"}
+    ...    {"name": "then","value": "then"}
+    ...    {"name": "evaluate","value": "then"}
```

Comment:
Do we need this many members in a test? Understand there should be more than one or two and testing aliases is a good idea, but several dozens feels too much.

Reply:
that is just an Enum that is used by the only library that uses enums and it has funny enum Members.

So i just took it.
I can delete some if it is a problem.